src/MSI/Product.wxs

```diff
@@ -78,6 +78,7 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
             <File Id="Microsoft.Identity.Client.dll" Source="Microsoft.Identity.Client.dll" />
             <File Id="Microsoft.Identity.Client.Extensions.Msal.dll" Source="Microsoft.Identity.Client.Extensions.Msal.dll" />
             <File Id="Microsoft.IdentityModel.Abstractions.dll" Source="Microsoft.IdentityModel.Abstractions.dll" />
+            <File Id="Microsoft.Bcl.TimeProvider.dll" Source="Microsoft.Bcl.TimeProvider.dll" />
```

Comment:
@v-rakeshsh Please update this in alphabetical order.

Reply:
Hi @v-viyada made changes as per suggested review comments, please re-review.
Thanks